specs/_features/maxeb_increase/capella.py

```diff
@@ -656,6 +658,8 @@ class BeaconState(Container):
     # Registry
     validators: List[Validator, VALIDATOR_REGISTRY_LIMIT]
     balances: List[Gwei, VALIDATOR_REGISTRY_LIMIT]
+    activation_validator_balance: Gwei
```

Comment:
Suggest keeping things symmetric, e.g. naming it `activation_queue_churn` to mirror `exit_queue_churn`.

Reply:
so the semantics here are slightly different. `exit_queue_churn` tracks the amount of Gwei exited for the current max exit epoch; `activation_validator_balance` tracks the amount of Gwei that the current validator at the front of the activation queue has activated. 

it feels to me like they are quite different values, so naming them to mirror eachother might be less accurate ? wdyt

Reply:
nitpick, but new state variables should be appended at the end to not break proof paths unnecessarily